.github/workflows/bump_support_rotation.yml

```diff
@@ -0,0 +1,23 @@
+name: bump_support_rotation
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Mondays at 2pm UTC (9am EST)
+    - cron: "0 14 * * 1"
+
+jobs:
+  bump_support_rotation:
+    runs-on: ubuntu-latest
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v2
+      - name: Bump Lyft Support Rotation
+        run: ./tools/bump_lyft_support_rotation.sh
+      - name: Commit Changes
+        run: |
+          git config --global user.email "${GITHUB_ACTOR}"
+          git config --global user.name "${GITHUB_ACTOR}@users.noreply.github.com"
+          git add .github/lyft_maintainers.yml
+          git commit -am "Bump Lyft Support Rotation"
+          git push
```

Comment:
The `main` branch is protected so this may need to open a PR and assign the person on rotation.

Reply:
Done in 9c50111.